### ISSUE TITLE
Change of hash function between v0.7 and v0.8

### ISSUE THREAD

**Issue by nponiros** [mmkal/slonik-tools#351]
The hash function for the migrations was changed in v0.8 and it now returns the name of the file instead of a hash of the file's content. Is there a reason for this change?

Looking at the code for v0.8 I would assume (didn't test it yet) that for all migrations already in the database a warning will be issued because the hash in the DB doesn't match the hash calculated for the file. Do we need to adjust all hashes already in the DB or is this a bug that should be fixed by using the old hash function?

**Comment by mmkal** [comment]
Yeah, this is a bug I'm afraid, mentioned also in #339. I will at some point push a change fixing the hashes to actually use the file content, but I may need to also add a utility for force-fixing hashes of already-run migrations. In the meantime, I'd suggest implementing your own hash function as described in the linked issue.
